### PR TITLE
new_sensor_type_with_ZOC

### PR DESCRIPTION
**Pull request by kellyhw**

### CODE REVIEW

**Review by shambergoldstein — APPROVED** [review]

File: ontology/yaml/resources/HVAC/entity_types/SENSOR.yaml [hunk 1]
```diff
@@ -34,6 +34,24 @@ SENSOR_ZTM_ZHM_CO2M_VOCM_PMDM_SPLM_ZAQIM_ZSPM_ZWVDM:
   uses:
   - illuminance_sensor
 
+  SENSOR_ZTM_ZHM_CO2M_VOCM_PMDM_SPLM_ZAQIM_ZSPM_ZOC:
```
Comment (shambergoldstein) [inline ontology/yaml/resources/HVAC/entity_types/SENSOR.yaml:37]: please fix tabbing here

**Review by shambergoldstein — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/HVAC/entity_types/SENSOR.yaml (hunk 1 above)
Comment (kellyhw) [reply ontology/yaml/resources/HVAC/entity_types/SENSOR.yaml:37]: Could you provide more clarification? 
Should ZOC be added to SENSOR_ZTM_ZHM_CO2M_VOCM_PMDM_SPLM_ZAQIM_ZSPM_ZWVDM existing entity type?

Comment (shambergoldstein) [reply ontology/yaml/resources/HVAC/entity_types/SENSOR.yaml:37]: I just mean that the canonical type name is indented/has extra spacing before it

Comment (kellyhw) [reply ontology/yaml/resources/HVAC/entity_types/SENSOR.yaml]: Hi @shambergoldstein , thanks for getting back to me on that.

**Pull request merged**